app/Http/Controllers/Campaign/WebhookController.php

```diff
@@ -79,14 +79,14 @@ public function store(StoreWebhook $request, Campaign $campaign)
         Webhook::create($data);
 
         return redirect()->route('webhooks.index', $campaign)
-            ->with('success_raw', __('campaigns.webhooks.create.success'));
+            ->with('success_raw', __('campaigns/webhooks.create.success'));
```

Comment:
You only need to use success_raw when you have a link in the success message. Otherwise just `success` is fine